dowhy/causal_estimator.py

```diff
@@ -225,6 +227,34 @@ def estimate_effect_naive(self):
         est = np.mean(df_withtreatment[self._outcome_name]) - np.mean(df_notreatment[self._outcome_name])
         return CausalEstimate(est, None, None, control_value=0, treatment_value=1)
 
+    def effect_tt(self, df: pd.DataFrame, *args, **kwargs):
+        """
+        Effect of treatment on the treated
+        @param df: unit features and treatment values
+        @param args: passed through to effect estimator
+        @param kwargs: passed through to effect estimator
+        @return: np.array of len(df) with effects of the actual treatment applied
+        """
+
+        eff = self.effect(df, *args, **kwargs).reshape((len(df), len(self._treatment_value)))
+
+        out = np.zeros(len(df))
+        if is_sequence(self._treatment_value) and not isinstance(self._treatment_value, str):
```

Comment:
an easier way to do this is to use parse_state from `from dowhy.utils.api import parse_state`
you can simply write, `self._treatment_value = parse_state(self._treatment_value)`

Reply:
`parse_state` doesn't handle `int` treatment values, would you like me to rewrite it to emulate the above behavior?